x-pack/plugins/uptime/server/lib/requests/index.ts

```diff
@@ -26,6 +26,11 @@ import { getJourneyFailedSteps } from './get_journey_failed_steps';
 import { getLastSuccessfulCheck } from './get_last_successful_check';
 import { getJourneyScreenshotBlocks } from './get_journey_screenshot_blocks';
 import { getSyntheticsMonitor } from './get_monitor';
+import {
+  getSyntheticsEnablement,
+  deleteServiceApiKey,
+  generateAndSaveServiceAPIKey,
+} from '../synthetics_service/get_api_key';
```

Comment:
Is it okay to add these to libs? Historically, libs has only included es requests. I feel like the lines are starting to blur. It's super convenient to add to libs, but should we keep some sort of structure here?

Reply:
I am fine with it.